components/eamxx/src/physics/rrtmgp/CMakeLists.txt

```diff
@@ -164,6 +164,8 @@ set (RRTMGP_TABLES
   scream/init/rrtmgp-allsky.nc
   scream/init/rrtmgp-cloud-optics-coeffs-sw.nc
   scream/init/rrtmgp-cloud-optics-coeffs-lw.nc
+  scream/init/rrtmgp-data-sw-g224-2018-12-04.nc
```

Comment:
Why are we adding these files too?  They seem unrelated to the PR.

Reply:
Right, this is a file requested by generate baseline. For machines with all the correct files, no error will be thrown since it just checks that the file is there then read it.

Reply:
okay sounds good, thanks for checking.